index.js

```diff
@@ -6,7 +6,7 @@ const fsp = require('fs').promises
 const path = require('path')
 
 async function main () {
-  const items = (process.env.INPUT_REPOSITORIES || '').split(/[\r\n]+/).filter(Boolean)
+  const items = (process.env.INPUT_REPOSITORIES || '').split(/[\s]+/).filter(Boolean)
```

Comment:
The brackets are no longer needed:

```suggestion
  const items = (process.env.INPUT_REPOSITORIES || '').split(/\s+/).filter(Boolean)
```